pilot/pkg/model/context.go

```diff
@@ -655,3 +657,12 @@ func (node *Proxy) GetInterceptionMode() TrafficInterceptionMode {
 
 	return InterceptionRedirect
 }
+
+// Inbound capture listener capture all port mode only supports iptables REDIRECT
+func (node *Proxy) IsInboundCaptureAllPorts() bool {
+	if node.GetInterceptionMode() != InterceptionRedirect {
+		return false
+	}
+	_, ok := node.Metadata[IstioInboundCaptureAllPort]
+	return ok
```

Comment:
Where is the value check? if I set the environment variable to false, this will still return true.
And what do you mean by InboundCaptureAllPort ?

Reply:
hmm.. reading more, this should be called UseSeparateVirtualInboundListener() [and rename the metadata variable accordingly, as its very cryptic currently].

Reply:
I recently learnt that the goal of in/out listener split is to support the feature that inbound_listener should capture all(vs specify container port). 
So split is just a implementation while IstioInboundCaptureAllPort is the interface.